src/components/card-link/index.tsx

```diff
@@ -33,14 +33,21 @@ const CardLink = ({
 			 * everything in the <a href>"
 			 *
 			 * https://adrianroselli.com/2020/02/block-links-cards-clickable-regions-etc.html
+			 *
+			 * Re: `span` with &nbsp; — Safari will only focus on links
+			 * that have content. This markup allows focus to behave as
+			 * expected while retaining the desired sibling
+			 * 'empty link' structure
 			 */}
 			<Link
 				aria-label={ariaLabel}
 				className={s.anchor}
 				data-heap-track={`card-link ${dataHeapTrack ?? ''}`}
 				href={href}
 				target={target}
-			/>
+			>
+				<span>&nbsp;</span>
```

Comment:
Should we set `aria-hidden` on this `span` for good measure? 🤔

Reply:
oh yes! somehow I removed that in my testing process. Will add back 😄